metadata-transformer/src/test/java/no/unit/nva/metadata/service/testdata/ValidDoiArgumentsProvider.java

```diff
@@ -1,32 +1,64 @@
-                Arguments.of("https://doi.org/bwfc", URI.create("https://doi.org/10.7774/CEVR.2016.5.1.1")),
-                Arguments.of("https://doi.org/bwfc/", URI.create("https://doi.org/10.7774/CEVR.2016.5.1.1")),
-                Arguments.of("http://doi.org/bwfc", URI.create("https://doi.org/10.7774/CEVR.2016.5.1.1")),
-                Arguments.of("http://doi.org/bwfc/", URI.create("https://doi.org/10.7774/CEVR.2016.5.1.1"))
+                Arguments.of(DC_IDENTIFIER, "https://doi.org/10.1109/5.771073",
+                        URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(CITATION_DOI, "https://doi.org/10.1109/5.771073",
+                        URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(DC_IDENTIFIER, "http://doi.org/10.1109/5.771073",
+                        URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(CITATION_DOI, "http://doi.org/10.1109/5.771073",
+                        URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(DC_IDENTIFIER, "https://dx.doi.org/10.1109/5.771073",
+                        URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(CITATION_DOI, "https://dx.doi.org/10.1109/5.771073",
+                        URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(DC_IDENTIFIER, "http://dx.doi.org/10.1109/5.771073",
+                        URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(CITATION_DOI, "http://dx.doi.org/10.1109/5.771073",
+                        URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(DC_IDENTIFIER, "10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(CITATION_DOI, "10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(DC_IDENTIFIER, "doi:10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(CITATION_DOI, "doi:10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(DC_IDENTIFIER, "doc:10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(CITATION_DOI, "doc:10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(DC_IDENTIFIER, "DOI:10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(CITATION_DOI, "DOI:10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(DC_IDENTIFIER, "DOC:10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(CITATION_DOI, "DOC:10.1109/5.771073", URI.create("https://doi.org/10.1109/5.771073")),
+                Arguments.of(DC_IDENTIFIER, "https://doi.org/bwfc",
```

Comment:
I do not want to be pedantic, but the short dois seem to me to be a separate case for testing.

Reply:
I have split this test to reflect what we are testing, hopefully the names now reflect this.